src/components/input/InputComponent.tsx

```diff
@@ -14,7 +14,7 @@ export const InputText:React.FC<InputProps> = (props) => (
         style={inputStyles.inputContainer}
         placeholder={props.placeholder}
         placeholderTextColor='#FFF'
-        value={props.input}
+        value={props.value}
```

Comment:
we have input property in InputProps interface but not value prop
so you using value that are not exist.